changelog/pr-log.js

```diff
@@ -50,20 +50,37 @@ const parseUpgradeNotes = p => {
 
 const loadPulls = async (
     since,
-    branches = 'master'
+    branches = 'master',
+    isDashboards
 ) => {
     let page = 1;
     let pulls = [];
+    let lastTagSha;
 
     const tags = await octokit.repos.listTags({
```

Comment:
Then this can integrate the tag argument from the new code below and we avoid dashboard-specific code.